shogun-lib/src/main/java/de/terrestris/shogun/lib/graphql/GraphQLDataFetchers.java

```diff
@@ -85,7 +85,7 @@ public DataFetcher getAllFiles() {
     }
 
     // ImageFile
-    @GraphQLQuery(name = "getImageFileByUuid")
+    @GraphQLQuery(name = "imageFileByUuid")
```

Comment:
You missed to add the change to the schema file. :)

Reply:
true